instance/ind_instance.go

```diff
@@ -324,6 +324,7 @@ func (inst *IndependentInstance) handleTaskDone(taskBehavior model.TaskBehavior,
 	}
 
 	if err != nil {
+		err = fmt.Errorf("handle task [%s] done in flow [%s] error: %s", taskInst.Name(), inst.flowDef.Name(), err.Error())
```

Comment:
maybe something like "error handling `task done` event for task [%s] in flow [%s] : %s"